lib/truffle/date.rb

```diff
@@ -277,11 +277,11 @@ def <=> (other)
       when Infinity; return d <=> other.d
       when Numeric; return d
       else
-	begin
-	  l, r = other.coerce(self)
-	  return l <=> r
-	rescue NoMethodError
-	end
+	       begin
```

Comment:
This looks wrong. Are there tabs in the file?

Reply:
There are.